server/core/src/https/views/errors.rs

```diff
@@ -58,6 +58,7 @@ impl IntoResponse for HtmxError {
                         UnrecoverableErrorView {
                             err_code: inner,
                             operation_id: kopid,
+                            domain_info: None,
```

Comment:
I don't know what I think about this one, because this is the only place we use domain_info as a "none". So could we give this a unique error view instead? But also if the point of this change is to render domain info on errors, this seems like the place where a lot of errors pass through, so we'd want the domain info here too wouldn't we?

Reply:
Sorted